Library/Formula/poppler.rb

```diff
@@ -52,7 +52,9 @@ def install
       --enable-introspection=yes
     ]
 
-    if build.with? "qt"
+    if build.with?("qt") && build.with?("qt5")
+      odie "poppler: --with-qt and --with-qt5 cannot be used at the same time"
```

Comment:
Thanks, will change that! Just out of curiosity: What is the difference? Because the result is (as far as I can tell) the same in both cases.